pyproject.toml

```diff
@@ -11,9 +11,9 @@ case-converter = "1.1.0"
 click = "8.0.4"
 click-default-group = "1.2.2"
 setuptools = "70.0.0"
-splight-lib = "^5.11.4"
+splight-lib = "^5.12.0"
```

Comment:
Use `poetry add` to update the dependencies version, this way poetry also updates the poetry.lock file